zpls_echograms/README.md

```diff
@@ -1,104 +1,193 @@
-| -dn, --deployment_number | Deployment number |  
 | -dd, --deployed_depth    | The depth where the ZPLSC/G is located at |  
-| -cr, --colorbar_range    | Set colorbar range. Useage: "min max" |  
-
-| **Required Input:** |  |  
-| ------------------- |----------------------------------- |
-| data_directory      | The path to /DATA directory, where the .01A files are stored |  
-| xml_file            | The path to .xml file |  
-| dates               | The date range to be plotted. E.g.: |  
-|                     | "20200118" will plot the day (01/18/2020), |  
-|                     | "202001" will plot the whole month (01/01/2020 to 01/31/2020), |  
-|                     | "202001 202002" will plot (01/01/2020 to 02/29/2020), |  
-|                     | "20200115 20200216" will plot (01/15/2020 to 02/16/2020) |  
-| output_dir          | The path where .nc file and .png plot is being saved |  
-
-**Note:** The preset colorbar ranges are the same for every array. Feel free to change it in the code.  
-
-**ZPLS raw data folder structure (This is how the instrument stores the data):**  
+| -cr, --colorbar_range    | Set colorbar range. Usage: "min max" |  
+| -vr, --vertical_range    | Set the range for the y-axis. Usage: "min max" |  
+
+| **Required Inputs:** |  |  
+| ------------------------- | ----------------------------------- |
+| -s, -site_code            | The OOI 8-letter site name for where the ZPLSC/G is located. Ex: "CP03ISSM", "GI02HYPM_Upper", or "GP02HYPM_Lower" |  
+| -d, --data_directory      | The path to the root data directory, below which the .01A or .raw files may be found |  
+| -o, --output_directory    | The path to the root data directory where .nc file and .png plot will be saved |  
+| -dr, --date_range         | The date range to be plotted, e.g.: |  
+|                           | "20200118" will plot the day (01/18/2020), |  
+|                           | "202001" will plot the whole month (01/01/2020 to 01/31/2020), |  
+|                           | "202001 202002" will plot (01/01/2020 to 02/29/2020), |  
+|                           | "20200115 20200216" will plot (01/15/2020 to 02/16/2020) |  
+| -zm, --zpls_model         | Instrument type, either an EK60 for cabled data, or an AZFP for uncabled |
+| -xf, --xml_file           | The path to .xml file to use if the instrument type is an AZFP |  
+
+**Notes:** 
+
+* The root data directory depends on the data source. If looking at cabled data and the EK60, the root data directory 
+  is the year (e.g. `2017/`). For uncabled data, the root data directory is `DATA/`.
+ 
+* The path to the `.xml` file is required if the ZPLS model is an AZFP. The `.xml` file contains instrument specific
+  settings and calibration coefficients needed to process the data.
+
+* In the case of the global moorings, the OOI 8-letter site name is appended with either an "_Upper" or "_Lower" to 
```

Comment:
In the case of the global **Apex Profiler (subsurface)** moorings

Reply:
Changes made